apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/V1/JwtTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright 2020 Google LLC
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     https://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax.ResourceNames;
+using Google.Apis.Auth.OAuth2;
+using Google.Cloud.ClientTesting;
+using Google.Cloud.Spanner.Data.CommonTesting;
+using Google.Cloud.Spanner.Data.IntegrationTests;
+using Grpc.Auth;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace Google.Cloud.Spanner.Admin.Instance.V1.IntegrationTests
+{
+    [Collection(nameof(SpannerDatabaseFixture))]
+    public class JwtTests
+    {
+        private readonly SpannerDatabaseFixture _fixture;
+
+        public JwtTests(SpannerDatabaseFixture fixture) => _fixture = fixture;
+
+        [Fact]
+        public void JwtAccessToken_GrpcClient()
+        {
+            InstanceAdminClientBuilder builder = new InstanceAdminClientBuilder
+            {
+                // Making sure we have no scopes so that a JWT is sent.
```

Comment:
Can't you just use `Scopes = new string[0]` in the builder? I believe that would achieve the same thing, but in a simpler way.